goodtablesio/integrations/s3/tasks/jobconf.py

```diff
@@ -36,32 +36,29 @@ def get_validation_conf(bucket, job_id):
         aws_access_key_id=access_key_id,
         aws_secret_access_key=secret_access_key)
 
-    # Get goodtables.yml contents (if it exists)
+    # Get job conf text (if it exists)
     try:
-        yml_file = client.get_object(
-            Bucket=bucket_name, Key='goodtables.yml')
-
-        job_conf = parse_job_conf(yml_file['Body'].read())
-
-    except botocore.exceptions.ClientError as e:
-        if e.response['Error']['Code'] == 'NoSuchKey':
-            job_conf = {'files': '*'}
+        yml_file = client.get_object(Bucket=bucket_name, Key='goodtables.yml')
+        job_conf_text = yml_file['Body'].read()
+    except botocore.exceptions.ClientError as exception:
```

Comment:
Add the `if e.response['Error']['Code'] == 'NoSuchKey'` bit so we don't catch other exceptions